### PR TITLE
[TACHYON-935] Added the usage of links in the documentation in the tachyon.security packages

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-935

### CODE REVIEW

File: common/src/main/java/tachyon/security/authentication/PlainSaslServer.java [hunk 1]
```diff
@@ -213,7 +213,7 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
   /**
    * An instance of this class represents a client user connecting to Tachyon service.
    *
-   * It is maintained in a ThreadLocal variable based on the Thrift RPC mechanism.
+   * It is maintained in a {@link ThreadLocal} variable based on the Thrift RPC mechanism.
```
Comment (aaudiber) [inline common/src/main/java/tachyon/security/authentication/PlainSaslServer.java]: For standard library classes like `ThreadLocal`, would it make sense to only `@link` them the first time they are mentioned in a file? There's no benefit to being able to rename them with an IDE.

**Comment by aaudiber** [comment]
Thanks for improving the docs and making them more resilient to renames! I left one comment

File: common/src/main/java/tachyon/security/authentication/PlainSaslServer.java (hunk 1 above)
Comment (HorizonNet) [inline common/src/main/java/tachyon/security/authentication/PlainSaslServer.java]: @aaudiber The idea is a little bit different and does not have something to do with refactoring in the first place. I link them the first time they are mentioned in the documentation for a method. The reason is that if someone reads the docs and wants to jump right into that class without scrolling to the top of the file and search for the place where the class is mentioned the first time.

Comment (aaudiber) [inline common/src/main/java/tachyon/security/authentication/PlainSaslServer.java]: sgtm, thanks for explaining

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**